execution_workflows/IsoSCM/modules/isoscm_processes.nf

```diff
@@ -0,0 +1,56 @@
+/*
+ * IsoSCM module containing all three steps in IsoSCM program:
+ *    - Assemble
+ *    - Enumerate (optional)
+ *    - Compare
+ */
+
+
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+
+process ISOSCM_ALL {
+        publishDir "${params.outdir}", mode: params.publish_dir_mode
+        
```

Comment:
It would be nice if you could include the Docker image URL here with: `container 'image URL'`. If you do not have this, you can create an issue that creates an image on the APAeval dockerhub.

Reply:
done